CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ cmake_policy(SET CMP0020 OLD)
 project(Molsketch)
 
 # Include the extra modules (OpenBabel/uninstall)
-set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/modules)
+set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake-modules)
```

Comment:
I think CMAKE_MODULE_PATH should be removed. It is overwritten by the environment.